lib/github-package.js

```diff
@@ -71,6 +72,7 @@ export default class GithubPackage {
           this.setActiveContext(WorkdirContext.absent());
         }
       }),
+      ContextMenuInterceptor,
```

Comment:
The `ContextMenuInterceptor` uses a single native event listener attached to the `document` that's shared among all instances. Each instance registers and deregisters an element in a Map, but the event listener itself isn't removed until the static `ContextMenuInterceptor.dispose()` method is called. This bit :point_up: ensures that we don't leak listeners on package deactivation.

... Although, this might be overkill. I _could_ just register a new document listener for each instance and check `element.includes(event.target)`? I was intending to minimize the number of listeners we register but maybe a single listener that does an O(n) operation on a Map isn't any better.